examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>drop the assertion test?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I was trying to explicitly show to the user what is equal to what. I can replace it with a print, but I think it would look clumsier. What do you think?</p>